pyincore/dfr3service.py

```diff
@@ -395,10 +395,17 @@ def _convert_properties_to_dict(rules, properties):
         matched_properties = {}
         # Handle legacy rules
         if isinstance(rules, list):
+            # If the rules are empty, return the matched properties
+            if not rules:
+                return matched_properties
```

Comment:
I think if rules are empty, we should match building to that curve.
@navarroc do you recall if that was that the design?

Reply:
Yes, if there are no rules, then it  should be a match. Also, you can test this by running test_roaddamage.py, which still fails for me.

Reply:
Can you give me some examples. I don't know how to make match building to the curve part.

Reply:
I think you make make line 399 be:

if not rules or rules == [None]:

Right now you are only checking if the list is empty. I think it's not empty, but contains "None" in this case.

Reply:
Oh I got it. I modified it. I tested with the road damage and it passed the part of this line but didn't go until the end since the test file was removed from dev, like you mentioned in other PR.

Reply:
Can you re-upload the galveston roads and update the ID in the test?

Reply:
We can probably use the one from production for galveston and copy it to dev. Looks like there are two of them.

Reply:
@navarroc I only see this one 606dd68fbe94522d1cc034b2 but it is a csv file. Is this correct? Or can you specify dataset id in here?

Reply:
On Prod: 5f15d04f33b2700c11fc9c4e